### ISSUE TITLE
1.7 broke everything

### ISSUE THREAD

**Issue by utkarshkukreti** [drbrain/meme#21]
Just updated to 1.7, getting this error for everything (`meme`, `meme --list`, ..)

```
D:/system/ruby/ruby/lib/ruby/gems/1.9.1/gems/meme_generator-1.7/lib/meme.rb:38:in `advice_dog': wrong number of arguments (4 for 3) (ArgumentError)
        from D:/system/ruby/ruby/lib/ruby/gems/1.9.1/gems/meme_generator-1.7/lib/meme.rb:70:in `<class:Meme>'
        from D:/system/ruby/ruby/lib/ruby/gems/1.9.1/gems/meme_generator-1.7/lib/meme.rb:9:in `<top (required)>'
        from <internal:lib/rubygems/custom_require>:29:in `require'
        from <internal:lib/rubygems/custom_require>:29:in `require'
        from D:/system/ruby/ruby/lib/ruby/gems/1.9.1/gems/meme_generator-1.7/bin/meme:3:in `<top (required)>'
        from D:/system/ruby/ruby/bin/meme:19:in `load'
        from D:/system/ruby/ruby/bin/meme:19:in `<main>'
```

**Comment by drbrain** [comment]
![no tests](http://i.imgur.com/t2tgT.jpg)

Fixed!  Try 1.7.1

**Comment by utkarshkukreti** [comment]
![Awesome](http://i.imgur.com/tpG40.jpg)
